x-pack/plugins/uptime/public/components/overview/monitor_list/monitor_list_drawer/enabled_alerts.tsx

```diff
@@ -16,6 +24,15 @@ interface Props {
   loading: boolean;
 }
 
+const LinkGroupList = styled(EuiListGroup)`
+  &&& {
```

Comment:
Didn't realise you could bump specificity using `&&&` (or any number of) with styled-components 👍